src/iocmme.cpp

```diff
@@ -804,6 +826,26 @@ void CmmeInput::CreateOriginalText(pugi::xml_node originalTextNode)
     return;
 }
 
+void CmmeInput::CreateProport(pugi::xml_node proportNode)
+{
+    assert(m_currentContainer);
+
+    /// Proportion part coming from CMME <Proportion>. In this case, create an MEI <proport> element is created alone
+    /// (not following an MEI <mensuration> element)
+    Proport *proport = new Proport();
+    int numVal = this->ChildAsInt(proportNode, "Num");
+    int denVal = this->ChildAsInt(proportNode, "Den");
+    if (numVal != VRV_UNSET) {
+        proport->SetNum(numVal);
+    }
+    if (denVal != VRV_UNSET) {
+        proport->SetNumbase(denVal);
```

Comment:
Verovio doesn't currently support the `SetNumbase()` method in the `Proport` object.